test/runtests.jl

```diff
@@ -374,7 +372,7 @@ end
         if !endswith(f, ".tar.gz") && !endswith(f, ".sha256")
             continue
         end
-        rm(f; force=true)
+        Base.rm(f; force=true)
```

Comment:
Is `Base.rm()` not exported anymore?!

Reply:
It is, but there's a conflict since `Pkg` moved to the `stdlib`, so now you do `using Pkg`, but `Pkg` exports it's own `rm` method, which conflicts w/ `Base.rm`. Not ideal, but it'll be solved by 1.0 since `Pkg` should go the way of the dodo.